decidim-admin/spec/shared/manage_process_steps_examples.rb

```diff
@@ -97,7 +98,7 @@
     let!(:process_step2) { create(:participatory_process_step, participatory_process: participatory_process) }
 
     before do
-      visit decidim_admin.participatory_process_path(participatory_process)
+      visit current_path
```

Comment:
I don't understand your doubts here. `current_path` is from Capybara, [see the docs](http://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession%3Acurrent_path). Returns the path of the current URL. What's confusing about this? What's redundant? :/